cpp/test/Slice/errorDetection/NotType.ice

```diff
@@ -2,29 +2,29 @@
 
```

Comment:
This whole test was strange. I reworked it to better test mistaken uses of non-types.
And removed the unused `ModuleN`s and commented out classes...